util/util.go

```diff
@@ -57,7 +57,7 @@ var (
 )
 
 func init() {
-	clusterv1.AddToScheme(scheme.Scheme)
+	_ = clusterv1.AddToScheme(scheme.Scheme)
```

Comment:
:grimacing: Is this something we should be doing in a utility method meant to be imported by other things? @vincepri @ncdc I think we might want to document this as a requirement and let comsumers handle this. thoughts?

Reply:
Yes, we need to remove this. Also, `scheme.Scheme` is the scheme for client-go for the full set of built-in Kubernetes APIs (by default). No harm in using that, but it should be up to the caller, not something a util package does for you.

Reply:
@tahsinrahman Does removing this line affect anything?

Reply:
I'm fine if we do this as a followup, it just stood out as I was reviewing.

Reply:
created https://github.com/kubernetes-sigs/cluster-api/issues/1604 to track